parquet-common/src/main/java/org/apache/parquet/io/OutputFile.java

```diff
@@ -31,4 +33,5 @@
 
   long defaultBlockSize();
 
+  Path getPath();
```

Comment:
I don't know if I overlooked this one before or it is a new change. The module `parquet-column` should not depend on hadoop. That's why we have the separate module `parquet-hadoop`. We already have struggling issues that parquet-mr cannot be used without hadoop, let's not make it worse.

Reply:
apologies, this is a recent change done due to my suggestion; making `common` dependent on `hadoop` is indeed not a good idea. @shangxinli - I've checked the hadoop Path and FileSystem code, and it looks like having a Path class here is not a must. A regular String would do, eg `String getPath` in this interface (`OutputFile`). 
Then the ParquetWriter can use the `new Path(String)` call.